app/helpers/kuroko2/job_definitions_helper.rb

```diff
@@ -34,5 +34,59 @@ def star_link_for(definition)
         )
       end
     end
+
+    def format_kuroko_script(script_text)
+      raw(
+        script_text.each_line.map { |line|
+          formatted_line = ERB::Util.html_escape(line)
```

Comment:
[IMO] If `match[:type]` exists, this escape is unnecessary and bad for performance. How about this?

```rb
matched = Kuroko2::Workflow::ScriptParser::LINE_REGEXP.match(line.chomp)
case matched && matched[:type]
when 'wait'
  format_wait_task(line, matched)
when 'sub_process'
  format_sub_process_task(line, matched)
else
  ERB::Util.html_escape(line)
end
```